### ISSUE TITLE
dtype rgb16 plotting error

### ISSUE THREAD

**Issue by magnunor** [hyperspy/hyperspy#2226]
With the most recent pypi version of HyperSpy (1.5), hyperspy_gui_traitsui (1.2) and hyperspy_gui_ipywidgets (1.2), the following gives an error at the last line (`plot()`)

```python
import numpy as np
import hyperspy.api as hs

rgb_array = np.random.random(size=(100, 100, 3))
signal_rgb = hs.signals.Signal1D(rgb_array*(2**16-1))
signal_rgb.change_dtype("uint16")
signal_rgb.change_dtype("rgb16")
signal_rgb.plot()
```

The error:

```python
hyperspy/drawing/image.py in update(self, data_changed, **kwargs)                                                           
    400             vmin, vmax = utils.centre_colormap_values(vmin, vmax)
    401         else:
--> 402             vmin, vmax = vmin, vmax
    403
    404         if self.norm == 'auto' and self.gamma != 1.0:

UnboundLocalError: local variable 'vmin' referenced before assignment
```

**Comment by magnunor** [comment]
Just to make sure this wasn't due to an old library, I ran this with the latest version of all the dependencies. Still getting the same error message.

**Comment by ericpre** [comment]
This is a change in hyperspy introduced in https://github.com/hyperspy/hyperspy/pull/2216.

If I remember correctly, plotting rgb signal as colored images are not explicitly supported, so it worked before it was more or less by chance!

**Comment by magnunor** [comment]
hehe, things unexpectedly working is always nice :) (Even if they break later)

The documentation will be have to updated: http://hyperspy.org/hyperspy-doc/current/user_guide/tools.html?highlight=rgb16#changing-the-data-type

This also breaks some of my DPC signal visualization in pixStem, but I can remove that from the short course notebook.

Having the RGB plotting is quite a useful feature though, so getting it working again at some point would nice (but no rush!).

**Comment by ericpre** [comment]
Well, clearly I did not remember well since the following from the documentation is fairly clear:

> In addition to all standard numpy dtypes, HyperSpy supports four extra dtypes for **RGB images **for visualization purposes only**: rgb8, rgba8, rgb16 and rgba16. This includes of course multi-dimensional RGB images.

**Comment by francisco-dlp** [comment]
@magnunor, congratulations for finding the first v1.5 bug! (I think that it's not the first time that you have this honor.)

I think that this is an important regression. I am for fixing it and releasing 1.5.1 today or tomorrow (afterwards I'll be unavailable). What do you think?

**Comment by francisco-dlp** [comment]
I've fixed it. I'll send a PR soon.

**Comment by francisco-dlp** [comment]
Fixed in #2230

**Comment by ericpre** [comment]
#2230 is merged.
